package.json

```
@@ -16,10 +16,12 @@
     "grunt": "^1.5.2",
     "grunt-contrib-cssmin": "^4.0.0",
     "grunt-contrib-uglify": "^5.2.1",
+    "grunt-contrib-watch": "^1.1.0",
     "load-grunt-tasks": "^5.1.0"
   },
   "scripts": {
     "grunt": "grunt",
+    "grunt:watch": "grunt watch",
```

```suggestion
    "build": "grunt",
    "watch": "grunt watch",
```

Let's simplify the scripts a bit.